README.md

```diff
@@ -1,20 +1,42 @@
 # The Mapzen iOS SDK
 [![Circle CI](https://circleci.com/gh/mapzen/ios.svg?style=shield&circle-token=158f79f566b88fb913ad153ee8b00681112eb5a2)](https://circleci.com/gh/mapzen/ios)
 
+<p align=center>
+<img width="311" height="552" src="https://mapzen-assets.s3.amazonaws.com/images/ios-sdk-beta/sdk-beta-map.png">
```

Comment:
oooOOOOoooo new tab bar fancy! Can you push a new screenshot to s3 and update here?